Fitness/Controllers/ProBioViewController.swift

```diff
@@ -0,0 +1,248 @@
+//
+//  ProBioViewController.swift
+//  Fitness
+//
+//  Created by Yana Sang on 5/19/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ProBioViewController: UIViewController {
+
+    // MARK: - Private view vars
+    private let backButton = UIButton()
+    private let collectionView = UICollectionView(frame: .zero, collectionViewLayout: StretchyHeaderLayout())
+
+    // MARK: - Private data vars
+    private var pro: ProBio!
+    private var sections: [Section] = []
+
+    private enum Constants {
+        static let proBioBiographyCellIdentifier = "proBioBiographyCellIdentifier"
+        static let proBioExpertiseCellIdentifier = "proBioExpertiseCellIdentifier"
+        static let proBioHeaderViewIdentifier = "proBioHeaderViewIdentifier"
+        static let proBioLinksCellIdentifier = "proBioLinksCellIdentifier"
+        static let proBioRoutinesCellIdentifier = "proBioRoutinesCellIdentifier"
+    }
+
+    // MARK: - Private classes/enums
+    private struct Section {
+        var items: [ItemType]
+    }
+
+    private enum ItemType {
+        case biography
+        case expertise
+        case routines
+        case links
+    }
+
+    // MARK: - Custom Initializer
+    init(pro: ProBio) {
+        super.init(nibName: nil, bundle: nil)
+        self.pro = pro
+        self.sections = [
+            Section(items: [.biography, .expertise, .routines, .links])
+        ]
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updatesStatusBarAppearanceAutomatically = true
+        view.backgroundColor = .white
+
+        let edgeSwipe = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(back))
+        edgeSwipe.edges = .left
+        view.addGestureRecognizer(edgeSwipe)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        navigationController?.isNavigationBarHidden = true
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        switch UIApplication.shared.statusBarStyle {
+        case .lightContent:
+            backButton.setImage(UIImage(named: "back-arrow"), for: .normal)
+        case .default:
+            backButton.setImage(UIImage(named: "darkBackArrow"), for: .normal)
+        }
+    }
+
+    // MARK: - Targets
+    @objc func back() {
+        if let navigationController = navigationController {
+            navigationController.popViewController(animated: true)
```

Comment:
can just do this in one line
`navigationController?.popViewController(animated: true)`